e2e/components/tabs/tabs.e2e.ts

```diff
@@ -77,8 +77,17 @@ function getFocusStates(elements: ElementArrayFinder) {
  * @param elements
  * @returns {webdriver.promise.Promise<Promise<boolean>[]>|webdriver.promise.Promise<T[]>}
  */
-function getActiveStates(elements: ElementArrayFinder) {
-  return getClassStates(elements, 'md-tab-active');
+function getLabelActiveStates(elements: ElementArrayFinder) {
+  return getClassStates(elements, 'md-tab-label-active');
+}
+
+/**
+ * Returns an array of true/false that represents the active states for the provided elements
+ * @param elements
+ * @returns {webdriver.promise.Promise<Promise<boolean>[]>|webdriver.promise.Promise<T[]>}
```

Comment:
Since this is TypeScript, you don't need the type in the JsDoc (these were formerly js)